Doc/library/platform.rst

```diff
@@ -40,9 +40,12 @@ Cross Platform
    and then only if the executable points to the Python interpreter.  Reasonable
    defaults are used when the above needs are not met.
 
+   To only get the pointer size of the running Python interpreter,
```

Comment:
`sys.maxsize` gives you the size of `Py_ssize_t`. On most platforms this is the same as the pointer size, but it is not guaranteed.

But I do not think that this addition is needed. `platform.architecture()` without arguments give you the bittness of the running Python interpreter. It does not try to guess it by parsing the `file` output.